state-driver/README.md

```diff
@@ -278,4 +320,285 @@ function MainContainer() {
+import { useEffectorStore } from "./hooks/useEffectorStore";
+
+function RecommendationsCarousel() {
+  const recsState = useEffectorStore(GBISearchStateDriver.recommendations.recsOutputStore);
+  
+  // Everything you need is in one object
+  const { 
+    products,     // Current page products (what UI displays)
+    loading,      // Loading state
+    error,        // Error state
+    pagination,   // Pagination info
+    metadata      // Response metadata
+  } = recsState;
+
+  if (loading) return <div>Loading recommendations...</div>;
+  if (error) return <div>Error: {error}</div>;
+  if (!products.length) return <div>No recommendations found</div>;
+
+  return (
+    <div className="recommendations-carousel">
+      <h3>You might also like</h3>
+      <div className="products-grid">
+        {products.map(product => (
+          <ProductCard key={product.id} product={product} />
+        ))}
+      </div>
+      {pagination.totalPages > 1 && (
+        <PaginationControls 
+          currentPage={pagination.currentPage}
+          totalPages={pagination.totalPages}
+        />
+      )}
+    </div>
+  );
+}
+```
+
+### Using Vanilla JavaScript (Non-Module Environment)
+
+```javascript
+// Subscribe to recommendations state changes
+const unsubscribe = window.GBISearchStateDriver.recommendations.recsOutputStore.watch((state) => {
+  if (state.loading) {
+    document.getElementById('recs-container').innerHTML = '<div>Loading...</div>';
+  } else if (state.error) {
+    document.getElementById('recs-container').innerHTML = `<div>Error: ${state.error}</div>`;
+  } else {
+    renderRecommendations(state.products);
+  }
+});
+
+// Get current state
+const currentState = window.GBISearchStateDriver.recommendations.recsOutputStore.getState();
+```
+
+## 8.4. Pagination Controls
+
+The recommendations system includes built-in pagination functions:
+
+```javascript
+// Navigation functions
+GBISearchStateDriver.recommendations.nextPage();        // Go to next page
+GBISearchStateDriver.recommendations.previousPage();   // Go to previous page
+GBISearchStateDriver.recommendations.resetRecs();      // Reset to first page
+
+// Change page size
+GBISearchStateDriver.recommendations.setRecsPageSize(10);
+
+// Example pagination component
+function PaginationControls({ currentPage, totalPages }) {
+  const handleNext = () => {
+    if (currentPage < totalPages - 1) {
+      GBISearchStateDriver.recommendations.nextPage();
+    }
+  };
+
+  const handlePrevious = () => {
+    if (currentPage > 0) {
+      GBISearchStateDriver.recommendations.previousPage();
+    }
+  };
+
+  return (
+    <div className="pagination">
+      <button onClick={handlePrevious} disabled={currentPage === 0}>
+        Previous
+      </button>
+      <span>Page {currentPage + 1} of {totalPages}</span>
+      <button onClick={handleNext} disabled={currentPage >= totalPages - 1}>
+        Next
+      </button>
+    </div>
+  );
+}
+```
+
+## 8.5. Common Recommendation Use Cases
```

Comment:
*Recommendations, I think, for consistency/clarity.

This is a great section, actually.